src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+     action="/" 
+     method="GET"
+     data-qa="big"
+     class="form form--big"
+     > 
+      <label >
+        <img 
+          src="images/Search.svg" 
+          class="iconBig"
+          alt="search image"
+          >
```

Comment:
same here with closed tag
```suggestion
        <img 
          src="images/Search.svg" 
          class="iconBig"
          alt="search image"
        >
```